bumble/pandora/security.py

```diff
@@ -229,11 +229,14 @@ def __init__(self, device: Device, config: Config) -> None:
         self.config = config
 
         def pairing_config_factory(connection: BumbleConnection) -> PairingConfig:
+            identity_address_type = config.identity_address_type
```

Comment:
nit: `identity_address_type = PairingConfig.AddressType.PUBLIC if connection.self_address.is_public else config.identity_address_type` to avoid unnecessary double assignment